analysis_options.yaml

```diff
@@ -10,13 +10,8 @@ linter:
     - avoid_unused_constructor_parameters
     - avoid_void_async
     - cancel_subscriptions
-    - collection_methods_unrelated_type
```

Comment:
I suspect you're using your analysis options diff'ing tool here? I wonder if there's a way to roll that into `ecosystem/pkgs/blast_repo`? If so, perhaps worth tracking in an issue.